src/main/steps/c100-rebuild/c100sequence.ts

```diff
@@ -161,4 +162,9 @@ export const C100Sequence: Step[] = [
       return nextPage?.url || C100_CONFIDENTIALITY_DETAILS_KNOW;
     },
   },
+  {
+    url: C100_START,
```

Comment:
@kumarmadurai Can we please make this as very first page to show when we click on "Start Now" on dashboard ?